shell/gobin.sh

```diff
@@ -21,7 +21,7 @@ if [[ $1 == "-p" ]]; then
 fi
 
 if [[ -z $1 ]] || [[ $1 =~ ^(--help|-h) ]]; then
-  echo "Usage: $0 [-p|-h|--help] <package> [args]" >&2
+  echo "Usage: [BUILD_DIR=...|BUILD_PATH=...] $0 [-p|-h|--help] <package> [args]" >&2
```

Comment:
I'd be ok with just proxying any args given to gobin.sh over to the underlying gobin....  (instead of us adding these one by one :) )

Reply:
Or even do GOBIN_ARGS instead of individual args.